README.md

```diff
@@ -44,15 +42,18 @@ a hosted Kubernetes service like [GKE](https://cloud.google.com/kubernetes-engin
 Setting up Kong for Kubernetes is as simple as:
 
 ```shell
-# Using Helm
+# Install the Gateway API CRDs before installing Kong Ingress Controller.
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.8.1/standard-install.yaml
```

Comment:
I'd hold off with this one until v1.

OTOH though, I'm not sure if we'd like to put this verbatim here as we'll have to update this whenever a version of Gateway API that we support gets released (and corresponding KIC version is out). Perhaps we could leverage renovate for this here 🤔 ?

Reply:
I'll try to use Renovate, because `README.md:7` also references this version thus doing it automatically will be beneficial. I think the robust pattern is to set this version based on the version referenced in `go.mod`

Reply:
Now it has some corner cases (e.g. referencing a different version in the conformance badge than the installation guide) and we have a tight schedule. I bumped the version in the installation guide to `v1.0.0-rc2` and improved the acceptance criteria for issues
- https://github.com/Kong/kubernetes-ingress-controller/issues/4998
- https://github.com/Kong/kubernetes-ingress-controller/issues/4654
to not lose your idea. WDYT?